Tests/Translation/Extractor/TwigExtractorTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace PrestaShop\TranslationToolsBundle\Tests\Translation\Extractor;
+
+use PrestaShop\TranslationToolsBundle\Tests\PhpUnit\TestCase;
+use PrestaShop\TranslationToolsBundle\Translation\Extractor\TwigExtractor;
+use Symfony\Bridge\Twig\Extension\TranslationExtension;
+use Symfony\Component\Translation\MessageCatalogue;
+use Twig\Environment;
+use Twig\Loader\FilesystemLoader;
+
+class TwigExtractorTest extends TestCase
+{
+    /**
+     * @var TwigExtractor
+     */
+    protected $twigExtractor;
+    private $resourcesDir;
+
+    public function setUp(): void
+    {
+        $this->resourcesDir = $this->getResource('directory/twig/');
+        $templatesLoader = new FilesystemLoader($this->resourcesDir);
+        $twig = new Environment($templatesLoader);
+        $twig->addExtension(new TranslationExtension());
+        $this->twigExtractor = new TwigExtractor($twig);
+    }
+
+    public function testExtractFromDirectory()
```

Comment:
You should type hint all method you added, and add strict types everywhere. I let you check all your changes